packages/react/src/components/Slider/Slider.tsx

```diff
@@ -484,11 +637,12 @@ export default class Slider extends PureComponent {
    *   clientX is not provided.
    * @param {boolean} [params.useRawValue=false] `true` to use the given value as-is.
    */
-  calcValue = ({ clientX = null, value = null, useRawValue = false }) => {
+
+  calcValue = ({ clientX, value, useRawValue = false }: CalcValueProps) => {
     const range = this.props.max - this.props.min;
-    const boundingRect = this.element.getBoundingClientRect();
-    const totalSteps = range / this.props.step;
-    let width = boundingRect.right - boundingRect.left;
+    const boundingRect = this.element?.getBoundingClientRect();
```

Comment:
This will ensure `this.element` is not null, but should we also protect for if the method isn't found on `this.element`?

```suggestion
    const boundingRect = this.element?.getBoundingClientRect?.();
```

Reply:
Sounds good to me